### PR TITLE
Alchen1218/macos proxy notarization

### PR DESCRIPTION
**Pull request by alchen1218**

### CODE REVIEW

File: macos_proxy_notarization/proxy_notarization.sh [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+      exit 1
+    fi
+  done
+}
+
+main() {
+  check_notarized_list
+  create_dev_certs
+  parse_proxy_version_tarball
+  echo $VERSION
```
Comment (ajayj89) [inline macos_proxy_notarization/proxy_notarization.sh:127]: are we not passing the version as param from jenkins job?

File: .travis.yml [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+  - secure: iSO5fkgws2yZkNsPFmKbitufAxNrXTesQKYRo7ogh8SdCFlLvGJq/nT/vF0Z24vMJ+xbIr2V1Uc4nVAcY+1nYiQuWlk5ifiXXCQmo4iKNLmiPcqoU0d8BJTucxt4cupTTzhqiI/29h+EaJZ2fBIyXYVMkNsCXpTDiHcZYLXLhJgXifi/s/isAm8aJ44YpSFV7vvS+l6AtzNXURoRGR96+uVc+9nlH60Ac+88ULyN/Mv+T2Q6A/h4VPVLwdPQxRipzkoQpqtYhamPeNHr4aW9rRlP7k2BX8NcVBNFqYCFRtLyabRRyIpMOyQQ9TQFJPvrNA0TOjNGG7NV2ZIURkdtHx31W9nhtvLX8AP9Xcb+1fmuX7PeG06qkeeQ70rKgBLRfAVg0ZOw7/UcNg2BISBfjbv9MNMeUogzVAatUMDVeBnK7nX5H/41wHRNIaMjbc3nIZ8WtyhgQzzXD0CPXc73M8TfN/7m/pA4pz1LrlJIq9UDhIXUNL/UER6WhBA9YUUvhhebMmTMY8T8GLmrIhrRPdT/5cRRG2u2Mu1foQGOhM54VGNV3WtaIWml5Bf0wZ24t0emRX6jFELJdyJsfnhxjw/CyIUPhwd/sTnlrkJKU30/SMXbsBpYJ2KNrEvD7tYof3ubkP6kie5/IjH7wbAnVYLEUCAbKm2IrkLAc4sFKXY=
```
[the remark this answers is not in the document]
Comment (alchen1218) [reply .travis.yml:9]: Yeah, this is how it is done for the wf python sdk

File: macos_proxy_notarization/proxy_notarization.sh (hunk 1 above)
Comment (alchen1218) [reply macos_proxy_notarization/proxy_notarization.sh:127]: I'm passing in the tarball file name ie: "wfproxy-7.0.0.tar.gz" 
the variable, "VERSION" is the parsing the actual version number:7.0.0

**Comment by basilisk487** [comment]
Can we move this directory into pkg/ or proxy/?

**Comment by alchen1218** [comment]
@basilisk487 Yeah, I'll move it into proxy/ and update the jenkins job.

**Pull request merged**